server.mjs

```
@@ -35,8 +35,5 @@ if (isMainThread) {
   }
 
   context.setContext({ workDir, shouldUseTypeScriptParserForJS, sonarlint, debugMemory, bundles });
-
-  server
-    .start(Number.parseInt(port), host, createWorker(new URL(import.meta.url), getContext()))
-    .catch(() => {});
+  server.start(Number.parseInt(port), host, createWorker(new URL(import.meta.url), getContext()));
```

Consciously we remove the catch. There doesn't seem to be a benefit of silently swallowing the error.